pycbc/inference/option_utils.py

```diff
@@ -714,13 +714,21 @@ def add_plot_posterior_option_group(parser):
     pgroup.add_argument('--plot-marginal', action='store_true', default=False,
                         help="Plot 1D marginalized distributions on the "
                              "diagonal axes.")
+    pgroup.add_argument('--marg-percentiles', nargs='+', default=None,
```

Comment:
Hey this all looks good! Nice!

Though can we can change the command line option from ``--marg-percentiles`` to something like ``--marginal-percentiles``? The string ``marg`` isn't really commonly used anywhere else where instead ``mariginal`` is used.

Reply:
Somehow I suspected that would be an issue :D I usually favor shorter command option names, but sure I'll change it